docs/index.adoc

```diff
@@ -61,46 +63,161 @@ ____
+The XCP slave implementation is responsible for translating the addresses to the physical addresses needed to access the variable values.
 
-This standard forbids the use of the `including mechanism` of additional `A2L` files to simplify complete extraction and copying of `A2L` files.
+=== XCP Protocol Settings
 
-It is recommended to include `IF_DATA XCP` elements to help MCD tools to connect and interact with the XCP slave inside the FMU more reliably and without user interaction.
+The `A2L` description shall include `IF_DATA XCP` elements to help MCD tools to connect and interact with the XCP slave inside the FMU more reliably and without user interaction.
+Parts of the `IF_DATA_XCP` description depend on the machine where the FMU binary is executed, for example, the IP address and port.
 
-=== XCP Slave Startup and Shutdown
+The default IP address used by the FMU exporter shall be `localhost`, i.e. `127.0.0.1`, which fits in many cases.
+The requirement for the port number is, that it must be unique on the machine where the FMU binary is executed.
+Typically, a certain range of ports is reserved for this purpose.
+The FMU importer is responsible for checking if any conflicts of the defined IP addresses and port numbers occur in the context of the simulated system.
 
-It is recommended to start the XCP service during `fmi3EnterInitializationMode` (or earlier) and shut down during `fmi3Terminate` (or later) if the FMU has no explicit power-up signal to simplify user interactions between simulator and MCD tool.
+Sometimes it is necessary for the FMU importer to override the default IP address and/or port number which was assigned by the FMU exporter.
+Therefore, the FMU model description shall contain two structural parameters which are used by the XCP slave during the configuration phase to setup the connection for the XCP protocol, see <<XCP Slave Implementation>>.
 
+.XCP connection parameters
+[[figure-xcp-connection-parameters]]
+----
+    IPAddress
+        Description:  "IP address or host name of the machine where the FMU binary is executed"
+        Type:         String
+        Causality:    structuralParameter
+        Variability:  fixed
+        Start:        "127.0.0.1"
+
+    XcpPortNumber
+        Description:  "Port number where the XCP slave listens for XCP protocol commands"
+        Type:         UInt16
+        Causality:    structuralParameter
+        Variability:  fixed
+        Start:        <in the range of 32768 to 39999>
+----
+
+The consumer of an FMU is responsible for keeping all occurrences of the IP address and port number consistent, either by manual adjustment of the FMU model description and the A2L file, or with tool support.
+
+
+=== XCP Slave Implementation
+
+The XCP service shall be started during `fmi3ExitConfigurationMode` and shut down during `fmi3Terminate` if the FMU has no explicit power-up signal to simplify user interactions between simulator and MCD tool.
```

Comment:
Why `fmi3ExitConfigurationMode`? Isn´t the more natural time during `fmi3EnterInitializationMode`?
The prior assumes that structural parameters are set, but for many FMUs that is not required - but all FMUs must be moved through the Initialization Mode...